### PR TITLE
added cart page

### PR DESCRIPTION
**Pull request by Rohanmond**
Added cart Page
Without login it will directed to login page
you can increase decrease the cart number

preview=> https://fashi-fy-react-git-cartlist-rohanmond.vercel.app/

### CODE REVIEW

File: fashify/src/pages/Cart/CartList.jsx [hunk 1]
```diff
@@ -38,9 +43,9 @@ const CartList = () => {
               {cartData.map((el) => {
                 return (
                   <div className="mngmt_cart-price-item" key={el.id}>
-                    <p>{el.name}</p>
-                    <p>Q:{el.quantity}</p>
-                    <p>₹{el.price * el.quantity}</p>
+                    <p>{el.title}</p>
+                    <p>Q:{el.qty}</p>
+                    <p>₹{el.price * el.qty}</p>
```
Comment (manish591) [inline fashify/src/pages/Cart/CartList.jsx]: You are typing `el.something` many times, so you can destructuring here. But, if you like this approach you can stick with this.

**Review by manish591 — APPROVED** [review]
Everything is looking good, the code is readable. 
Here are some suggestions: 
* You import like `../../../../something`, I think this makes your imports look bad, so you can try barrel exports. using index.js file.

File: fashify/src/pages/Cart/CartList.jsx (hunk 1 above)
Comment (Rohanmond) [reply fashify/src/pages/Cart/CartList.jsx]: sure I will change those

**Review by shrey27 — APPROVED** [review]
- Overall code looks good
- Its good that you have made functions for filters, it makes them reusable
- Just a suggestion, Try to define all api end points at one place, so that if any of them changes, you can edit them inside one file

File: fashify/src/pages/Cart/CartList.jsx [hunk 2]
```diff
@@ -15,12 +20,12 @@ const CartList = () => {
...
-              <Cart
+              <CartCard
                 el={el}
                 key={el.id}
-                removeCartHandler={removeCartHandler}
-                quantityIncreaseHandler={quantityIncreaseHandler}
-                quantityDecreaseHandler={quantityDecreaseHandler}
+                // removeCartHandler={removeCartHandler}
+                // quantityIncreaseHandler={quantityIncreaseHandler}
+                // quantityDecreaseHandler={quantityDecreaseHandler}
```
Comment (anuj-kumary) [inline fashify/src/pages/Cart/CartList.jsx]: Remove unwanted comment

File: fashify/src/pages/Cart/components/CartCard/CartCard.jsx [hunk 3]
```diff
@@ -13,7 +68,7 @@ const Cart = ({
       <div className="card-content">
         <div className="cart_mngmt-card-container">
           <div className="cart_mngmt-card-item">
-            <h4>{name}</h4>
+            <h4>{title}</h4>
           </div>
           <div className="cart_mngmt-card-item">
             <p className="font-wt-semibold">₹{price}</p>
```
Comment (anuj-kumary) [inline fashify/src/pages/Cart/components/CartCard/CartCard.jsx]: Give space between price and rupees symbol

File: fashify/src/DataReducer/constants.js [hunk 4]
```diff
@@ -4,6 +4,7 @@ export const ActionType = {
   ChangeFilter: "change-filter",
   ClearFilter: "clear-filter",
   SetWishList: "set-wishlist",
+  SetCartList: "set-cartlist",
```
Comment (anuj-kumary) [inline fashify/src/DataReducer/constants.js:7]: Try to give action type in CAPITAL letters

**Review by anuj-kumary — APPROVED** [review]

**Pull request merged**